docs/application/dotnet/getting-started/sec-privileges.md

```diff
@@ -0,0 +1,191 @@
+|----------------------------------------|------|------|----------------------------------------|
+| `http://tizen.org/privilege/internet`    | public |  | Most of the mobile and wearable devices use a cellular network for IP communication. However, the cellular network can cause data costs and an application that sends data through the Internet can be crucial for user privacy. Due to the importance of the functionality, a privilege for controlling application Internet access has been added.<br><br>The new privilege is coupled with IP addresses of the destination and source of the IP packets. If your socket is connecting or listening to any IP address except 127.0.0.1, this privilege is required to communicate properly. If your application does not have this privilege, the connection is blocked in the kernel layer and returns an error in the `connect()` function as the permission is denied. If you are listening to a socket, you never receive any packets from the outside without errors on the socket functions.<br><br>If you are using the `listen()` and `connect()` functions between the local loopback interface (127.0.0.1), you cannot connect to a random application (due to sandboxing) no matter how you add this privilege. However, you can connect between multiple processes of the same application binary. |
+| `http://tizen.org/privilege/mediastorage` | public | Storage | When you connect the device to a computer (Windows&reg; or macOS) through USB, you can access a dedicated media storage area shown as massive media storage. This region of the storage is called media storage and is usually used for multimedia files, such as photos, videos, and music files. Since this storage area is used for user private data, access to it must be protected with a privilege.<br>If your application does not have this privilege, no file operations into the media storage area succeed and you receive a permission denied error. If you have this privilege, you can read and write directories and files, create new files, and delete files in the storage area.<br><br>This privilege is treated as privacy privilege since platform version 4.0. |
+| `http://tizen.org/privilege/externalstorage` | public | Storage | Similar to the media storage, many devices support external storages, such as MicroSD card or USB memory. As with the media storage, the access to an external storage must be protected with a privilege. You can find the absolute path of the external storage with the [Storage](../../api/mobile/latest/group__CAPI__SYSTEM__STORAGE__MODULE.html) API functions, such as `storage_get_root_directory()` function.<br>If your application does not have this privilege, all file operations fail with a permission denied error. If you have this privilege, you have full access to the external storage.<br><br>This privilege is treated as privacy privilege since platform version 4.0. |
```

Comment:
Please check the hyperlink for its landing page, seems to be throwing an error.

Reply:
done